source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1358,12 +1358,16 @@ It will be refreshed by the next periodic check or,
 if an application operation needs the server sooner than that,
 then a re-check will be triggered by the server selection algorithm.
 
-Transient Server Errors
-```````````````````````
+"not master" and "node is recovering"
+`````````````````````````````````````
+These errors are detected from a getLastError response,
+write command response, or query response. Clients MUST check if the server
+error is a "node is recovering" error or a "not master" error.
+
+If the response includes an error code, it MUST be used to determine if the
+error message is a "node is recovering" or "not master" error.
 
-Server errors are detected from a getLastError response, write command
-response, or query response. The following errors indicate a server
-role change:
+The following error codes indicate a "node is recovering" error:
```

Comment:
Let's say, "The following error codes indicate a replica set member is temporarily unusable. These are called "node is recovering" errors:"

When I first read this I thought it's weird to call InterruptedAtShutdown a synonym for "node is recovering," I think the new sentence clarifies.

Reply:
Done.